test/controllers/likes_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+# require "test_helper"
+
+# class LikesControllerTest < ActionDispatch::IntegrationTest
```

Comment:
uh oh! commented out tests! Should make tests when user creates or destroys likes

Reply:
working on them!

Reply:
done!